packages/odyssey-react-mui/src/Tooltip.tsx

```diff
@@ -49,3 +49,5 @@ export const Tooltip = ({
     <MuiPropsChild>{children}</MuiPropsChild>
   </MuiTooltip>
 );
+
+Tooltip.displayName = "Tooltip";
```

Comment:
Can we also add a memoized export with `memo` like our other components?

Reply:
Done.